CONTRIBUTING.md

```diff
@@ -375,6 +376,22 @@ The circleci uses pytest-split to balance unittest load on multiple nodes. In or
 CI_PYTEST_SPLIT_ARGS="--store-durations" tox -e py37
 ```
 
+### Functional Testing
+
+TODO: overview of how to write and run functional tests with [yea](https://github.com/wandb/yea)
+and the [yea-wandb](https://github.com/wandb/yea-wandb) plugin.
+
+The `yea-wandb` plugin for `yea` uses copies of several components from `tests/utils` 
+(`artifact_emu.py`, `mock_requests.py`, and `mock_server.py`) 
+to provide a test environment for functional tests. Currently, we maintain a copy of those components in 
+`yea-wandb/src/yea_wandb`, so they need to be in sync.
+
+If you update one of those files, you need to:
+- Make a new branch in `yea-wandb` with the new versions of the files.
+- Point the client branch you are working on to this `yea-wandb` branch.
+- Merge and release `yea-wandb`.
```

Comment:
Once you are happy with your changes to the above files need to do the following steps.